### PR TITLE
James pre-release readthrough

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]

File: content/02.introduction.md [hunk 1]
```diff
@@ -47,20 +47,20 @@ archaeologists, museum curators, and scientists can also contribute 'modern'
...
 techniques to overcome these challenges.
 
 Two previously published and commonly used pipelines in the field are PALEOMIX
 [@doi:10.1038/nprot.2014.063] and EAGER [@doi:10.1186/s13059-016-0918-z]. These
 two pipelines take a similar approach to link together standard tools used for
 Illumina NGS data processing (sequencing quality control, sequencing adapter
-removal and/or paired-end read merging, mapping of reads to a reference genome,
-genotyping, etc.), but with a specific focus on tools that are designed for or
-well-suited for aDNA (such as bwa aln that works well on ultra-short molecules
+removal/and or paired-end read merging, mapping of reads to a reference genome,
```
Comment (TCLamnidis) [inline content/02.introduction.md:61]: Reverting correction to a typo

File: content/03.results-and-discussion.md [hunk 2]
```diff
@@ -44,17 +44,17 @@ ensure other groups can generate the same results. Usage of profiles also
 reduces mistakes caused by insufficient 'prose' based reporting of program
 settings that can be regularly found in the literature. The default
 nf-core/eager profile uses parameters evaluated in different aDNA specific
```
Comment (TCLamnidis) [inline content/03.results-and-discussion.md:46]: aDNA-specific

File: content/03.results-and-discussion.md [hunk 3]
```diff
@@ -137,14 +137,14 @@ bacterial genome reconstruction [@doi:10.1146/annurev-genom-091416-035526]. The
 output SNP alignment FASTA file can then be used for downstream analyses such as
 phylogenetic tree construction. Simple coverage statistics of particular
 annotations (e.g. genes) of an input reference is offered by bedtools
-[@doi:10.1093/bioinformatics/btq033], which can be used, for example, for
+[@doi:10.1093/bioinformatics/btq033], which can be used in cases such as for
```
Comment (TCLamnidis) [inline content/03.results-and-discussion.md:140]:
```suggestion
[@doi:10.1093/bioinformatics/btq033], which can be used in various cases, such as for
```

Small change, but somehow it reads better to me with an extra word there.

File: content/04-conclusion.md [hunk 4]
```diff
@@ -4,8 +4,8 @@ nf-core/eager is an efficient, portable, and accessible pipeline for processing
 ancient genomic data. This re-implementation of EAGER into Nextflow and nf-core
 will improve reproducibility and inclusion of rapidly increasing aDNA datasets,
 for both large and small laboratories. Extensive documentation also enables
-newcomers to the field get a practical understanding on how to interpret aDNA in
-the context of NGS data processing. Ultimately, nf-core/eager provides easier
+newcomers to the field to get a practical understanding on how to interpret aDNA
+in the context of NGS data processing. Ultimately, nf-core/eager provides easier
 access to the latest tools and routine screening analyses commonly used in the
-field, and sets up the pipeline for staying at the forefront of palaeogenetic
+field, and sets up the pipeline for remainind at the forefront of palaeogenetic
```
Comment (TCLamnidis) [inline content/04-conclusion.md:10]:
```suggestion
field, and sets up the pipeline for remaining at the forefront of palaeogenetic
```

File: content/02.introduction.md (hunk 1 above)
Comment (jfy133) [reply content/02.introduction.md:61]: I think that's fixed in your next one, so will leave this here.

File: content/05.methods.md [hunk 5]
```diff
@@ -68,39 +76,39 @@ utilised for different nf-core pipelines), so users need not write their own.
...
-merging of files, the pipeline can be supplied with a tabular
-separated value (TSV) file to enable such complex data handling. Both FASTQs and
-BAMs can be provided in this set up. FASTQs with the same library name and
-sequencing chemistry but sequenced across multiple lanes will be concatenated
-after AdapterRemoval and prior mapping. Libraries with the sample name and with
-the same UDG treatment, will be merged after deduplication. If libraries with
-the sample name have different UDG treatment, these will be merged after the
-aDNA modification stage (i.e. BAM trimming or PMDtools, if turned on), prior
-genotyping, as shown in in Figure {@fig:merging-files}.
+merging of files, the pipeline can be supplied with a tabu separated value (TSV)
```
Comment (TCLamnidis) [inline content/05.methods.md:83]:
```suggestion
merging of files, the pipeline can be supplied with a tab-separated values (TSV)
```

File: content/05.methods.md [hunk 6]
```diff
@@ -68,39 +76,39 @@ utilised for different nf-core pipelines), so users need not write their own.
...
+file to enable such complex data handling. Both FASTQs and BAMs can be provided
+in this set up. FASTQs with the same library name and sequencing chemistry but
+sequenced across multiple lanes will be concatenated after AdapterRemoval and
+prior mapping. Libraries with the sample name and with the same UDG treatment,
+will be merged after deduplication. If libraries with the sample name have
+different UDG treatment, these will be merged after the aDNA modification stage
+(i.e. BAM trimming or PMDtools, if turned on), prior genotyping, as shown in in
```
Comment (TCLamnidis) [inline content/05.methods.md:90]:
```suggestion
(i.e. BAM trimming or PMDtools, if turned on), prior genotyping, as shown in 
```

File: content/05.methods.md [hunk 7]
```diff
@@ -68,39 +76,39 @@ utilised for different nf-core pipelines), so users need not write their own.
...
 
 As Nextflow will automatically download files from URLs, profiles and/or TSV
-files can include links to publicly available data (e.g. the ENA FTP server).
-This assists in reproducibility as if profiles or TSV files are uploaded with a
-publication, a researcher wishing to re-analyse the data in the same way can use
-the exact settings and merging procedures in the original publication, without
-having to reconstruct this from prose.
+files can include links to publicly available data (e.g. the ENA FTP server
+[@url:ftp://ftp.ebi.ac.uk/pub/databases/ena/]). This assists in reproducibility
+as if profiles or TSV files are uploaded with a publication, a researcher
```
Comment (TCLamnidis) [inline content/05.methods.md:101]: I don't like this `as if` bit. Consider changing to `since if`?

Comment (TCLamnidis) [reply content/05.methods.md:101]: or `because if`

File: content/05.methods.md [hunk 8]
```diff
@@ -68,39 +76,39 @@ utilised for different nf-core pipelines), so users need not write their own.
...
+wishing to re-analyse the data in the same way can use the exact settings and
+merging procedures in the original publication, without having to reconstruct
```
Comment (TCLamnidis) [inline content/05.methods.md:103]:
```suggestion
merging procedures as in the original publication, without having to reconstruct
```

File: content/05.methods.md [hunk 9]
```diff
@@ -68,39 +76,39 @@ utilised for different nf-core pipelines), so users need not write their own.
...
+this from prose.
 
 #### Monitoring
 
-Users can either monitor their pipeline execution with the messages Nextflow
-prints to the console while running, or utilize projects such as Nextflow Tower
-[@url:https://tower.nf] to monitor their analysis pipeline during runtime.
+Users can either monitor their pipeline execution with real-time messages
+Nextflow prints to the console while running, or utilize projects such as
```
Comment (TCLamnidis) [inline content/05.methods.md:109]:
```suggestion
Nextflow prints to the console while running, or utilise projects such as
```
Haven't these been corrected already?

**Review by TCLamnidis — CHANGES_REQUESTED** [review]

**Pull request merged**